web/src/bootstrap_typeahead.ts

```diff
@@ -322,7 +322,7 @@ export class Typeahead<ItemType extends string | object> {
     }
 
     select(e?: JQuery.ClickEvent | JQuery.KeyUpEvent | JQuery.KeyDownEvent): this {
-        const val = this.values.get(the(this.$menu.find(".active")));
+        const val = this.values.get(this.$menu.find(".active")[0]!);
```

Comment:
If this can be `undefined`, then we should not use `!`; we should explicitly handle the `undefined`.

Reply:
Oh of course, sorry 🤦